Pcap++/header/PcapFilter.h

```diff
@@ -84,6 +88,52 @@ namespace pcpp
 		virtual ~GeneralFilter();
 	};
 
+	//Used in FilterTester
+	class RawPacket;
+
+	/**
+	 * @class FilterTester
+	 * This class can be loaded with a filter string and then can be used to verify the string is valid and check if a packet matches it<BR>
+	 */
+	class FilterTester : public GeneralFilter
+	{
+#if __cplusplus > 199711L || _MSC_VER >= 1800 //Maybe this can be 1600 for VS2010

```

Comment:
Is using smart pointers really necessary here? I try to avoid using them wherever possible because of the different implementations in different versions of C++

Reply:
Not required. I am just spoiled and accustomed to using latest C++. dumb pointer will suffice and I can allocate when used and kill with destructor.